integration/class-friends-feed-parser-activitypub.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * This is the class for integrating ActivityPub into the Friends Plugin.
+ *
+ * @since 0.14
+ *
+ * @package ActivityPub
+ * @author Alex Kirk
+ */
+namespace Activitypub;
+
+class Friends_Feed_Parser_ActivityPub extends \Friends\Feed_Parser {
+	const SLUG = 'activitypub';
+	const NAME = 'ActivityPub';
+	const URL = 'https://www.w3.org/TR/activitypub/';
+	private $friends_feed;
+
+	public function __construct( \Friends\Feed $friends_feed ) {
+		$this->friends_feed = $friends_feed;
+
+		\add_action( 'activitypub_inbox_create', array( $this, 'handle_received_activity' ), 10, 2 );
+		\add_action( 'activitypub_inbox_accept', array( $this, 'handle_received_activity' ), 10, 2 );
+		\add_filter( 'friends_user_feed_activated', array( $this, 'follow_user' ), 10 );
```

Comment:
I used the cron to send the activities asynchronous, to not slow down activities on the WordPress plattform.

* https://github.com/pfefferle/wordpress-activitypub/blob/master/includes/class-activitypub.php#L121
* https://github.com/pfefferle/wordpress-activitypub/blob/master/includes/class-activity-dispatcher.php

Reply:
Good point, I added that.